docker-assets/run-postgresql

```diff
@@ -12,6 +12,19 @@ function set_miq_role() {
   psql --command "ALTER ROLE \"${POSTGRESQL_USER}\" SUPERUSER LOGIN PASSWORD '${POSTGRESQL_PASSWORD}';"
 }
 
+# Include MIQ config overrides if directory present
+function include_miq_config() {
+  POSTGRESQL_CONFIG_DIR=${POSTGRESQL_CONFIG_DIR:-/etc/manageiq/postgresql.conf.d}
+  if [ -d "${POSTGRESQL_CONFIG_DIR}" ]; then
+    cat >> "$PGDATA/postgresql.conf" <<EOF
+# Begin ManageIQ configuration overrides:
+include_dir '${POSTGRESQL_CONFIG_DIR}'
```

Comment:
Do we need to check if this is already here to avoid having multiple entries in the config?